src/core/Notification/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { AlertProps } from "@material-ui/lab";
+import React from "react";
+import { ReactComponent as IconAlert } from "../../common/svgs/IconAlert.svg";
+import { ReactComponent as IconClose } from "../../common/svgs/IconClose.svg";
+import { ReactComponent as IconSuccess } from "../../common/svgs/IconSuccess.svg";
+import { StyledButton, StyledNotification } from "./style";
+
+export { AlertProps };
```

Comment:
Same here, maybe `NotificationProps` instead of `AlertProps`, in case `CalloutProps` and `NotificationProps` diverge from `AlertProps` down the road, and we won't need to introduce a breaking change 🙆‍♂️ !